src/screens/HomePage/HomePage.tsx

```diff
@@ -72,6 +73,12 @@ export default function HomePage() {
   const exploreSectionRef = useRef(null);
 
   const [menuOpen, setMenuOpen] = useState(false);
+  const [showCompareModal, setShowCompareModal] = useState(false);
+
+  const vaccinationsTableButtonOnClick = () => {
+    window.location.href = '#compare';
+    setShowCompareModal(true);
```

Comment:
So clicking "See all states" ends up showing metros. 😬   I think we need to somehow plumb the location scope into compare...

Alternatively, after seeing the 5 top / bottom counties I'm not sure how useful that will be to highlight...  We could (sorry) go back to making the vaccinations table always show states, and then change compare to default to states instead of metros, which I think is more valuable anyway.

Reply:
+1 to making compare default to states overall. im not sure im following the rest of the comment though